src/main/steps/common/task-list/components/tasklist/utils.ts

```diff
@@ -125,6 +128,33 @@ const taskListConfig = {
           },
         ],
       },
+      {
+        id: TaskListSection.YOUR_HEARING,
+        content: getContents.bind(null, TaskListSection.YOUR_HEARING),
+        show: () => true,
+        tasks: [
+          {
+            id: Tasks.VIEW_HEARING_DETAILS,
+            href: (caseData: Partial<CaseWithId>) => {
+              if (!caseData) {
+                return '/';
+              }
+
+              if (caseData?.state === State.CASE_HEARING) {
+                return APPLICANT_TASK_LIST_URL;
+              }
+            },
+            show: (caseData: Partial<CaseWithId>): boolean =>
```

Comment:
show condition should be based on caseData & hearing details.
We can something like this - caseData.hearingCollection && caseData.hearingCollection.length > 0